logger.go

```diff
@@ -128,6 +128,10 @@ func (l *Logger) log(depth int, debug bool, format string, args ...interface{})
 
 	h.HandleEvent(&s.e)
 
+	for i := range s.e.Args {
```

Comment:
I should comment, thanks for pointing it out. This is to ensure that the backing array doesn't hold pointers before it is placed back into the pool. 
I'll add a comment.